io.github.DonaldKellett.csnaketerm.yml

```diff
@@ -0,0 +1,41 @@
+app-id: io.github.DonaldKellett.csnaketerm
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+modules:
+  - 'shared-modules/python2.7/python-2.7.json'
```

Comment:
You should copy this file to your local repo and add `"cleanup": ["*"]`.

Reply:
Noted. Do I add the cleanup field at the top level of the YAML file?